src/assets/scripts/airport/Fix/FixModel.js

```diff
@@ -0,0 +1,114 @@
+import _cloneDeep from 'lodash/cloneDeep';
+import _uniqueId from 'lodash/uniqueId';
+import PositionModel from '../../base/PositionModel';
+
+/**
+ * Defines a navigational `FixModel`
+ *
+ * A `FixModel` can be used as part of a `StandardRoute` or as a naviagtional aid.
+ * Not all `FixModel`s are a part of a `StandardRoute`.
+ *
+ * @class FixModel
+ */
+export default class FixModel {
+    /**
+     * @for FixModel
+     * @constructor
+     * @param fixName {string}
+     * @param fixCoordinate {array}
+     * @param airportPosition {PositionModel}
+     */
+    constructor(fixName, fixCoordinate, airportPosition) {
+        if (!fixName || !fixCoordinate || !airportPosition) {
+            return;
+        }
+
+        /**
+         * Unigue string id that can be used to differentiate this model instance from another.
```

Comment:
As I read further, I guess the question is why `clonePosition` is needed. Why is that better to create a deep copy of the position than a simple getter that retrieves and returns the position?